skimage/transform/tests/test_steerable.py

```diff
@@ -0,0 +1,94 @@
+from numpy.testing import assert_equal, assert_array_equal,\
+    assert_array_almost_equal, run_module_suite
+from skimage.transform import steerable
+import numpy as np
+from skimage import img_as_float
+
+
+def test_steerable_shape():
+    im = np.random.randint(0, 255, (128, 128), dtype=np.uint8)
+    coeff = steerable.build_steerable(im)
+
+    assert_array_equal(coeff[0].shape, [128, 128])
+    assert_array_equal(coeff[-1].shape, [16, 16])
+
+    for i in range(1, 4):
+        assert_equal(len(coeff[i]), 4)
+        for j in range(4):
+            m = 2**(8 - i)
+            assert_array_equal(coeff[i][j].shape, np.array([m, m]))
+
+
+def test_different_orientation_height():
+    im = np.random.randint(0, 255, (113, 29), dtype=np.uint8)
+    coeff = steerable.build_steerable(im, height=3, nbands=6)
+    out = steerable.recon_steerable(coeff)
+
+    assert_array_almost_equal(img_as_float(im), out, decimal=2)
+
+
+def test_steerable_reconstruction_power_of_two():
+    im = np.random.randint(0, 255, (128, 128), dtype=np.uint8)
+    coeff = steerable.build_steerable(im)
+    out = steerable.recon_steerable(coeff)
+
+    assert_array_almost_equal(img_as_float(im), out, decimal=2)
+
+
+def test_steerable_reconstruction_symmetric():
+    im = np.random.randint(0, 255, (200, 200), dtype=np.uint8)
+    coeff = steerable.build_steerable(im)
+    out = steerable.recon_steerable(coeff)
+
+    assert_array_almost_equal(img_as_float(im), out, decimal=2)
+
+
+def test_steerable_reconstruction_asymmetric():
+    im = np.random.randint(0, 255, (113, 29), dtype=np.uint8)
+    coeff = steerable.build_steerable(im)
+    out = steerable.recon_steerable(coeff)
+
+    assert_array_almost_equal(img_as_float(im), out, decimal=1)
+
+
+def test_steerable_reconstruction_power_of_two_float():
+    im = np.random.uniform(0, 1, size=(128, 128))
+    im = im.astype(np.float)
+    coeff = steerable.build_steerable(im)
+    out = steerable.recon_steerable(coeff)
+
+    assert_array_almost_equal(img_as_float(im), out, decimal=2)
```

Comment:
What is in the tests included here just seems to verify primarily that roundtrip decomposition/reconstruction is correct.
When you say "all functions are unit tested vs. Matlab", do you mean that you did this offline as an independent verification?  It is possible to archive expected outputs from Matlab to use during testing.  We have some tests that do that sort of thing in PyWavelets (see, [here](https://github.com/PyWavelets/pywt/blob/master/pywt/tests/test_matlab_compatibility.py) for example), but I don't know that we need to go to that level of trouble here.  Visually the coefficients in the example look as one would expect and the round-trip seems okay, so I think that is probably enough.